core/app/models/spree/order_updater.rb

```diff
@@ -114,7 +114,6 @@ def recalculate_adjustments
       # http://www.boe.ca.gov/formspubs/pub113/
       update_promotions
       update_taxes
-      update_cancellations
```

Comment:
I'm thinking this may be a breaking change for stores that for some reason customized `update_cancellations`. The deprecation message would never show, since the method is not called anymore. So, I'm wondering if just deprecating would be a safer approach initially, and just remove it later... @mamhoff what's your opinion on this?

Reply:
As long as the thing goes away at some point. Sure, I'll leave it in.

Reply:
Actually... That makes our test suite extremely loud, to the point that we would not see any meaningful deprecation warnings. This warning is IMO not particularly meaningful, as I cannot imagine people having overridden this code path. If they have, they hopefully have at least one test for it where the deprecation message would show.

Reply:
I think that updating the description of the PR is enough, keeping in mind that whoever will release will take a look at the description to build the changelog. We might want to create a label if you think it's needed.

Reply:
@kennyadsl I agree the title change should suffice.
@mamhoff do you prefer to update the OR title personally or is it OK if we do it on your behalf?

Reply:
Please, do it on my behalf. You know best how you want this change to appear in the change log.